.travis.yml

```diff
@@ -37,6 +37,7 @@ before_install:
        --allow-external petsc --allow-unverified petsc \
        --allow-external petsc4py  --allow-unverified petsc4py \
        < requirements-git.txt"
+  - pip install pulp
```

Comment:
If you install PuLP, you probably also want to add `glpk-utils` at the apt dependencies.